Cargo.toml

```diff
@@ -9,5 +9,10 @@ crate-type = ["cdylib"]
 
 [dependencies]
 pyo3 = { version = "0.25.1", features = ["extension-module", "experimental-async"] }
-scylla = "1.3.1"
+#scylla = "1.3.1"
+scylla = { git = "https://github.com/paulinaczajkowska/scylla-rust-driver", branch = "main" }
 tokio = { version = "1.42.0", features = ["full"] }
+thiserror = "2.0.6"
+uuid = { version = "1.11.0", features = ["v4"] }
+#scylla-cql = "1.3.1"
+scylla-cql = { git = "https://github.com/paulinaczajkowska/scylla-rust-driver", branch = "main" }
```

Comment:
@Lorak-mmk We shouldn't encourage using private forks of Rust Driver for such purposes. IMO, we should either:
1. Have a fork of Rust Driver in this org and update it with relevant APIs when needed, and sync it with upstream / merge to upstream from time to time, or
2. Merge relevant APIs to upstream immediately.

Reply:
And why do you need scylla-cql?

Reply:
I agree with that fully. I invite you to propose a design for scylla-cql that allows this, we can then implement it in 2.0.